src/mainwindow.h

```diff
@@ -172,6 +172,7 @@ private slots:
     SimpleCrypt crypto;
     QDateTime currentNoteLastEdited;
     bool notifyAllExternalModifications;
+    bool setViewMode;
```

Comment:
I think `set` isn't a good prefix for a variable. Just `viewMode` might be better...